addons/purchase_requisition/tests/test_purchase_requisition.py

```diff
@@ -90,6 +90,13 @@ def test_03_blanket_order_rfq(self):
 
         # lazy reproduction of clicking on "New Quotation" act_window button
         po_form = Form(self.env['purchase.order'].with_context({"default_requisition_id": bo.id, "default_user_id": False}))
+
+        po_form.order_line.remove(0)
+        with po_form.order_line.new() as line:
+            line.product_id = self.product_09
+            line.product_qty = 5.0
+            line.price_unit = 21
+
```

Comment:
move this and the `po.button_confirm` to after the existing `assertEqual()`s since those test that the initial creation values are correct. You can add your original assertEqual that the PO is correctly confirmed afterwards too and it doesn't hurt to check again that the values are correctly reset to the blanket order's price_unit (if you re-add the product without setting a price_unit)

Reply:
I think it should be OK now